src/screens/OrganizationPeople/OrganizationPeople.tsx

```
@@ -1,178 +1,277 @@
- * It also provides the functionality to search the members, admins and users by their full name.
- * It also provides the functionality to remove the members and admins from the organization.
- * @returns JSX.Element which contains the list of members, admins and users of the organization.
- */
-function organizationPeople(): JSX.Element {
+import SortingButton from 'subComponents/SortingButton';
+import Avatar from 'components/Avatar/Avatar';
+import AddMember from './AddMember';
+
+const PAGE_SIZE = 10;
+interface ProcesssedRow {
+  _id: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)